### PR TITLE
Integrate styled components from Skill Version,Skill creator and NotFound

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #2003 

Changes: Integrate Styled components in Skill Version,Skill creator and NotFound. And change px to rem.
Surge Deployment Link: https://pr-2008-fossasia-susi-skill-cms.surge.sh

### CODE REVIEW

File: src/components/SkillCreator/SkillCreator.js [hunk 1]
```diff
@@ -59,118 +60,124 @@ let languages = [];
...
-    top: '-3px',
-    width: '35px',
-    height: '35px',
+    left: '-0.125rem',
+    top: '-0.1875rem',
+    width: '2.1875rem',
+    height: '2.1875rem',
     color: 'rgb(158, 158, 158)',
     cursor: 'pointer',
     display: window.innerWidth < 769 ? 'none' : 'inherit',
```
Comment (fragm3) [inline src/components/SkillCreator/SkillCreator.js]: Convert this to styled components

File: src/components/SkillCreator/SkillCreator.js [hunk 2]
```diff
@@ -59,118 +60,124 @@ let languages = [];
...
   },
   chevronButton: {
```
Comment (fragm3) [inline src/components/SkillCreator/SkillCreator.js]: This as well

**Comment by harsh-jindal** [comment]
@fragm3 Styled component styles are not being applied to material ui icons.

File: src/components/SkillCreator/SkillViews/CodeView.js [hunk 3]
```diff
@@ -21,36 +21,40 @@ import 'brace/theme/terminal';
...
-    padding: '0 25px',
-    fontSize: '14px',
-  },
-  button: {
-    display: 'flex',
-    marginRight: '30px',
-    alignItems: 'center',
-    cursor: 'pointer',
-  },
   icon: {
```
[the remark this answers is not in the document]
Comment (harsh-jindal) [reply src/components/SkillCreator/SkillViews/CodeView.js]: @fragm3 About eliminating inline styles. There are a few components and html elements which have only one inline style, so shall I create a styled component for those also?

[the remark this answers is not in the document]
Comment (fragm3) [reply src/components/SkillCreator/SkillViews/CodeView.js]: Inline would be better in that case

**Comment by fragm3** [comment]
@harsh-jindal Styled component is being used on the close icon in this PR itself. Explore more please, there are some issues with styled component and material UI being used together. But we can't ignore those components. The color will be applied to icon using: fill. We definitely need styled components on prop dependent, responsive components

**Comment by harsh-jindal** [comment]
@fragm3 Made the changes.

**Comment by fragm3** [comment]
@harsh-jindal finalize this for migration, please.

**Comment by harsh-jindal** [comment]
@fragm3 Done.

**Pull request merged**